.github/workflows/smoketest-all.yml

```diff
@@ -0,0 +1,200 @@
+name: Scheduled Release Smoke Test
+
+on:
+  schedule:
+    # Runs every other Monday at 9:00 AM UTC
+    - cron: '0 9 */14 * *'
```

Comment:
Have we decided on a day to run it, or is it more of a suggestion? I think Monday or Tuesday is good. Tuesday the latest to have time to act on potential problems!

Reply:
I think mabye we can run it every day?

Reply:
Sure!